src/olympia/stats/views.py

```diff
@@ -336,7 +337,7 @@ def check_stats_permission(request, addon, beta):
                 user.email.endswith('@mozilla.com') or
                 waffle.flag_is_active(request, 'beta-stats')
             )
-        )
+        ) or addon.type not in [ADDON_EXTENSION, ADDON_STATICTHEME]
```

Comment:
can you refactor this `if` out into some boolean variables to make the ending `if` more straightforward.